test/k8sT/manifests/guestbook_deployment.yaml

```diff
@@ -4,24 +4,24 @@ apiVersion: v1
 metadata:
   name: redis-master
   labels:
-    k8s-app.guestbook: redis
-    role: master
+    app: redis
+    role: leader
+    tier: backend
 spec:
   replicas: 1
   selector:
-    k8s-app.guestbook: redis
-    role: master
+    app: redis
   template:
     metadata:
       labels:
-        k8s-app.guestbook: redis
-        role: master
-        zgroup: guestbook
+        app: redis
+        role: leader
+        tier: backend
     spec:
       terminationGracePeriodSeconds: 0
       containers:
-      - name: redis-master
-        image: docker.io/library/redis:4.0.11
+      - name: leader
+        image: "docker.io/redis:6.0.5"
```

Comment:
`docker.io/library/redis`

Reply:
No preference either way, but trying to understand as Google takes different convention than you: why do you prefer the `docker.io/library/xxx` form?